packages/docusaurus-theme-bootstrap/src/theme/BlogListCard/index.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import BlogPostCard from '@theme/BlogPostCard';
+
+function BlogListCard(props) {
```

Comment:
Call this file and component `BlogListPage` to be consistent with `theme-classic`?

Reply:
Yeah, It's better. It was good to understand and know that is possible to change the name of the components, in case of have a own name more descriptive